webapp/src/widgets/menu/colorOption.tsx

```diff
@@ -12,8 +13,11 @@ type ColorOptionProps = MenuOptionProps & {
 
 function ColorOption(props: ColorOptionProps): JSX.Element {
     const {id, name, icon} = props
+    const intl = useIntl()
     return (
         <div
+            role='button'
+            aria-label={intl.formatMessage({id: 'BoardComponent.selectColor', defaultMessage: 'Select {color} Color'}, {color: name})}
```

Comment:
I think the id here should be `ColorOption.selectColor`, not `BoardComponent.selectColor`.